ui/components/providers/workflow/forms/update-via-credentials-form.tsx

```diff
@@ -96,9 +96,11 @@ export const UpdateViaCredentialsForm = ({
   const onSubmitClient = async (values: FormType) => {
     const formData = new FormData();
 
-    Object.entries(values).forEach(
-      ([key, value]) => value !== undefined && formData.append(key, value),
-    );
+    Object.entries(values).forEach(([key, value]) => {
```

Comment:
Please revert this change in this PR. I'm handling it in a new branch specifically for this.

Reply:
Without this the build is failing and was blocking Matt. Added it temporarily so the NextJS build happens.